src/frontend/src/session.rs

```diff
@@ -62,6 +62,8 @@ pub struct OptimizerContext {
     pub session_ctx: Arc<SessionImpl>,
     // We use `AtomicI32` here because  `Arc<T>` implements `Send` only when `T: Send + Sync`.
     pub next_id: AtomicI32,
+    /// For debugging purposes, store the SQL string in Context
+    pub sql: Arc<String>,
```

Comment:
We should prefer `Arc<str>` over `Arc<String>`.

Reply:
Thanks, updated to use `str` instead of `String`